williamchart/src/main/java/com/db/williamchart/data/LineChartConfiguration.kt

```diff
@@ -1,20 +1,22 @@
 package com.db.williamchart.data
 
 data class LineChartConfiguration(
-    override val width: Int,
-    override val height: Int,
-    override val paddings: Paddings,
-    override val axis: AxisType,
-    override val labelsSize: Float,
-    val lineThickness: Float,
-    val pointsDrawableWidth: Int,
-    val pointsDrawableHeight: Int,
-    val fillColor: Int,
-    val gradientFillColors: IntArray
+        override val width: Int,
+        override val height: Int,
+        override val paddings: Paddings,
+        override val axis: AxisType,
+        override val labelsSize: Float,
+        val lineThickness: Float,
+        val pointsDrawableWidth: Int,
+        val pointsDrawableHeight: Int,
+        val fillColor: Int,
+        val gradientFillColors: IntArray,
+        val xScaleLabel: (String) -> String = { it },
+        val yScaleLabel: (Float) -> String = { it.toString() }
```

Comment:
I'm thinking if it makes sense to have these as part of the `ChartConfiguration` since all charts will use it. So instead of making it part of `LineChartConfiguration` and `BarChartConfiguration` separately, having it only in `ChartConfiguration`.

Reply:
Agreed, fixed.